index.html

```diff
@@ -323,6 +330,7 @@ <h2 class="javaS">CSS</h2>
           </div>
         </div>
       </div>
+    
```

Comment:
- [ ] Your portfolio is looking great! Let's make it even better by adding the missing horizontal lines from the Figma template:

|Template|Your desgin|
|---|---|
|![Captura de pantalla 2022-06-01 a la(s) 9 15 45 p  m](https://user-images.githubusercontent.com/83717285/171538431-cccaf64e-7af2-4c15-8a30-70bbc5745f9a.png)|![Captura de pantalla 2022-06-01 a la(s) 9 15 23 p  m](https://user-images.githubusercontent.com/83717285/171538510-716d4dd6-2365-4e39-857b-f516aac4a5c4.png)|

Reply:
done